### ISSUE TITLE
2 types of melee weapons

### ISSUE THREAD

**Issue by Haox09** [deferred-impact/battledudes-issue-tracker#154]
The melee weapons in Battledudes.io just feels the same. Sure, every single one of them look different to the other ones, but when it comes to use they are identical. How about creating two types of melees? (fist&gloves not effected, basically third catagory, stays the same) To not focus on too much cosmetics and make the game unsurviv potentially, these are small changes.
The smaller and shorter melees, e.g. Knifes, karambits, will have a small attack rate buff, while having the range right now or smaller. The larger melees (swords, pans, axes, sabers, blades, bats, hammers, trident, saws etc.) will recieve a slower attack rate (reasonable because they are bigger and have a bigger swing) but a little further reach. If there are confusion on which weapons are considered "short" and which "long" they can all be discussed later. I feel like melee is a field of BD that needs a little sparkle. Nothing too fancy like a animation, just reach and attack rates.

**Comment by deferred-impact** [comment]
I wouldn't mind this, as long as both types are balanced (as far as you described, they would be) and anyone can reasonably easily acquire either type (i.e. buy it with coins).

I would also make it so that large melees deal damage in an arc, possibly hitting multiple enemies (which would finally make sense and resolve the animation hitbox dissonance), though of course that would require a large rate of fire nerf to compensate

**Comment by Gregory-Y** [comment]
Good idea, and when viewing melees in the store or inventory you should be able to see whether they are a short or long melee.

**Comment by deferred-impact** [comment]
> Good idea, and when viewing melees in the store or inventory you should be able to see whether they are a short or long melee.

I was gonna say that, but i got distracted and forgot. Thanks for having the same thought, lmao. I probably wouldn't've remembered

**Comment by deferred-impact** [comment]
Screenshot the poll pls

**Comment by Haox09** [comment]
<img width="498" alt="image" src="https://github.com/deferred-impact/battledudes-issue-tracker/assets/117471088/73f69b4c-e0f1-4c5f-be4a-2ef2d770eb7a">
